src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -113,7 +113,7 @@ public void test() throws IOException { //NOPMD - assert is done in subs.
 		playerDies();
 		clickPopup(DEATH_TOMAIN);
 
-		mainMenu(MAIN_QUIT);
+		Platform.runLater(stage::close);
```

Comment:
This change was really needed to make the E2E run, right? Please confirm, if so, then leave this be.

<review,testing